userappwithbroker/src/main/java/com/microsoft/aad/adal/example/userappwithbroker/Constants.java

```diff
@@ -60,7 +60,8 @@ public String getText() {
     enum ClientId {
         MSDEVEX("b92e0ba5-f86e-4411-8e18-6b5f928d968a"),
         ONEDRIVE("af124e86-4e96-495a-b70a-90f90ab96707"),
-        OFFICE("d3590ed6-52b3-4102-aeff-aad2292ab01c");
+        OFFICE("d3590ed6-52b3-4102-aeff-aad2292ab01c"),
+        BLACKFOREST("f5d01c1c-abe6-4207-ae2d-5bc9af251724");
```

Comment:
Can you rename to AppCheck2_BF or something like that, we might have more ClientId from them in the future

Reply:
Done 👍